src/agents/actions/index.ts

```
@@ -23,12 +23,21 @@ export const welcome = () => {
   };
 };
 
-async function interpretAction(text: string): Promise<Omit<EmailAction, 'id' | 'createdAt'>> {
-  const systemPrompt = `You convert user requests about email actions into JSON.\nReturn only JSON with fields action (notify|summarize), criteria, and description.`;
+async function interpretAction(
+  text: string,
+): Promise<Omit<EmailAction, 'id' | 'createdAt'>> {
+  const systemPrompt = `You convert user requests about email actions into JSON.\nReturn only JSON with fields event, action, description, optional criteria, and optional notify {method, frequency}.`;
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)